tests/helpers.py

```diff
@@ -61,6 +63,25 @@ def open(self, *args, **kwargs):
         return super(CTFdTestClient, self).open(*args, **kwargs)
 
 
+class FreezeTime(str, RawEnum):
+    NOT_STARTED = "2017-10-3"  # Tuesday, October 3, 2017
+    STARTED = "2017-10-5"  # Thursday, October 5, 2017
+    ENDED = "2017-10-7"  # Saturday, October 7, 2017
+
+
+class CTFtime:
+    def __enter__(self):
+        set_config(
+            "start", "1507089600"
+        )  # Wednesday, October 4, 2017 12:00:00 AM GMT-04:00 DST
+        set_config(
+            "end", "1507262400"
+        )  # Friday, October 6, 2017 12:00:00 AM GMT-04:00 DST
+
+    def __exit__(self, exc_type, exec_value, exec_traceback):
+        pass
```

Comment:
In this code block you should probably undo the above set_config calls

Reply:
I'm using `set_config("start", 0)` to achieve this right now, but I also saw a `clear_config()` function https://github.com/CTFd/CTFd/blob/master/CTFd/cache/__init__.py#L21. Is `clear_config()` the better approach to undo the value setting?

Reply:
You would probably want to do set_config("start", None)